Formula/mariadb@10.1.rb

```diff
@@ -24,7 +24,6 @@ class MariadbAT101 < Formula
   deprecated_option "with-tests" => "with-test"
 
   depends_on "cmake" => :build
-  depends_on "pidof" unless MacOS.version >= :mountain_lion
```

Comment:
You are right! But as already mentioned `pidof` is no longer required, they use `pgrep` now.